src/Website/Resources/order.php

```diff
@@ -30,10 +30,11 @@
     $_SESSION["email"] = $email;
 
     if ($link->query($sql) === TRUE) {
-           
+        $link->close();
```

Comment:
right! i remember us reading about having to remember to close the link and then we never did it. good catch!!